dynamiqs/time_array.py

```diff
@@ -227,24 +213,42 @@ def ndim(self) -> int:
         return len(self.shape)
 
     @abstractmethod
-    def reshape(self, *args: int) -> TimeArray:
-        """Returns a time-array containing the same data with a new shape.
+    def reshape(self, *new_shape: int) -> TimeArray:
+        """Gives a new shape to an array without changing its data.
 
         Args:
-            *args: New shape.
+            *new_shape: New shape.
 
         Returns:
             New time-array object with the given shape.
         """
 
+    @abstractmethod
+    def broadcast_to(self, *new_shape: int) -> TimeArray:
+        """Broadcast a time-array to a new shape.
+
+        Args:
+            new_shape: The new shape should be compatible with the original shape.
+
+        Returns:
+            New time-array object with the given shape.
+
+        """
+
     @abstractmethod
     def conj(self) -> TimeArray:
-        """Returns the element-wise complex conjugate of the time-array.
+        """Computes element-wise complex conjugate of the time-array.
 
         Returns:
             New time-array object with element-wise complex conjuguated values.
         """
 
+    @abstractmethod
```

Comment:
Nit: maybe this should also be a property?

Reply:
Fixed in #565.